.github/workflows/pr.yml

```diff
@@ -66,3 +67,10 @@ jobs:
           # use a fake infura key for the bundle size check so this test will
           # run successfully on external contributor Pull Requests
           INFURA_KEY: "badc0de0deadc0debadc0de0deadc0de"
+      - name: Upload artifact
```

Comment:
Checkout the Candidate artifact here: https://github.com/trufflesuite/ganache/actions/runs/3709596319